Classes/KIFUIViewTestActor.m

```diff
@@ -111,6 +111,14 @@ - (instancetype)usingTraits:(UIAccessibilityTraits)accessibilityTraits;
     return [self usingPredicate:predicate];
 }
 
+- (instancetype)usingAbsenceOfTraits:(UIAccessibilityTraits)accessibilityTraits;
+{
+    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"(accessibilityTraits & %@) != %@", @(accessibilityTraits), @(accessibilityTraits)];
```

Comment:
If a bitmask of `usingAbsenceOfTraits:(UIAccessibilityTraitButton | UIAccessibilityTraitSelected)` is supplied, must both traits be absent or if either trait is absent is that sufficient? I'm pretty sure this will match something that doesn't match either of the traits, not both of them. I think I'd expect neither trait to be present though, but I could be convinced maybe. The alternative would be `(accessibilityTraits & %@) == 0`.

Either way, we should make it explicit in the header comment about which behavior is expected. It would also be good to clarify in this methods header and `usingTraits:` header that `usingTraits:UIAccessibilityTraitNone` does nothing. If you don't mind adding that while you're in there, that would be great.

Reply:
We discussed and this is fine as it is. Just added some examples as to the expected behavior in the header.